swap_meet/clothing.py

```diff
@@ -1,2 +1,18 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, id=None, fabric=None, condition=0.0, age=0.0):
+        super().__init__(id, condition, age)
+
+        if not fabric:
+                fabric = "Unknown"
+        self.fabric = fabric
```

Comment:
[suggestion] You can use `"Unknown"` as your default param. Unlike `[]`, strings don't have the same problem where you can accidentally share the same object.

```python
    def __init__(self, id=None, fabric="Unknown", condition=0.0, age=0.0):
        super().__init__(id, condition, age)
        self.fabric = fabric
```